### ISSUE TITLE
Show count of multiple words selected

### ISSUE THREAD

**Issue by kaukawalaaziz** [richrace/highlight-selected#115]
Package already shows the count of highlighted similar words but it would be great if package could show the count of selected words. It will be boon for those who love using multiple cursors.

Somewhat like this:-
![image](https://cloud.githubusercontent.com/assets/10600699/13403451/19968fdc-df3b-11e5-89fc-2d8aab4d4a84.png)

**Comment by richrace** [comment]
Go to settings and check ago in status bar.
-- 
Richard Race

**Comment by richrace** [comment]
See here: 

![show_in_status_bar](https://cloud.githubusercontent.com/assets/1273965/13404747/134f4dc6-df13-11e5-9659-9b248614c8c2.png)

**Comment by kaukawalaaziz** [comment]
yes it does shows the count of matched words but it should also show the count of words selected.

Example: 
I have 10 matched words but I only select 5 by using Ctrl + D.

So it should also show the count of selected words i.e 5

**Comment by richrace** [comment]
Ohhh ok. I understand now. Thanks.

**Comment by kaukawalaaziz** [comment]
@richrace Welcome :)
